tutorons/tests/css/test_query.py

```diff
@@ -8,17 +8,125 @@
 from bs4 import BeautifulSoup
 from django.test import Client
 
-
 logging.basicConfig(level=logging.INFO, format="%(message)s")
 
 
+class FetchExplanationsForMatthewsCSSSelectorTest(unittest.TestCase):
```

Comment:
Name this something like "FetchExplanationsForSelectorsInStylesheetTest".  We can leave our names out of the methods :)